packages/lib/OgImages.tsx

```diff
@@ -236,3 +241,18 @@ export const Generic = ({ title, description }: GenericImageProps) => (
     </div>
   </Wrapper>
 );
+
+export const ScreenShot = ({ image }: ScreenshotImageProps) => (
+  <Wrapper rotateBackground>
+    <div tw="h-full w-full flex flex-col justify-center items-center">
+      <img
+        src={image}
+        width="1024"
+        height="576"
+        alt="screenshot"
+        tw="rounded-2xl mt-[140px] object-cover"
+        style={{ boxShadow: "0 0 45px -3px rgba(0,0,0,.3)" }}
```

Comment:
The Satori TW property doesn't support a custom box shadow, that's why there's an inline style declaration here.